### PR TITLE
fix: fix ci error

### PR DESCRIPTION
**Pull request by googs1025**
#### What this PR does / why we need it
fix ci error
#### Which issue(s) this PR fixes

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by googs1025** [comment]
@kerthcet /PTAL I didn't think clearly when I submitted this pr. I'm sorry.

File: pkg/controllers/leaderworkerset_controller.go [hunk 1]
```diff
@@ -91,26 +97,23 @@ func (r *LeaderWorkerSetReconciler) Reconcile(ctx context.Context, req ctrl.Requ
...
 	if err := r.SSAWithStatefulset(ctx, lws, partition); err != nil {
-		r.Record.Eventf(lws, corev1.EventTypeWarning, constants.SSALeaderWorkerSetFailedReason, err.Error())
 		return ctrl.Result{}, err
 	}
 
 	// Create headless service if it does not exist.
 	if err := r.createHeadlessServiceIfNotExists(ctx, lws); err != nil {
 		log.Error(err, "Creating headless service.")
-		r.Record.Eventf(lws, corev1.EventTypeWarning, constants.HeadlessServiceCreationFailedReason, err.Error())
+		r.Record.Eventf(lws, corev1.EventTypeWarning, HeadlessServiceCreationFailedReason, err.Error())
```
Comment (kerthcet) [inline pkg/controllers/leaderworkerset_controller.go:111]:
```suggestion
		r.Record.Eventf(lws, corev1.EventTypeWarning, HeadlessServiceCreationFailedReason, fmt.Sprintf("Failed to create headless service for error: %v", err))
```

File: pkg/controllers/leaderworkerset_controller.go [hunk 2]
```diff
@@ -63,6 +62,13 @@ const (
 	fieldManager = "lws"
 )
 
+const (
+	// HeadlessServiceCreationFailedReason
+	// Event reason used when a Headless Service creation fails.
+	// The event uses the error(s) as the reason.
+	HeadlessServiceCreationFailedReason = "HeadlessServiceCreationFailed"
```
Comment (kerthcet) [inline pkg/controllers/leaderworkerset_controller.go:69]: "FailedCreate" let's make it simple, we'll see the object at the column `OBJECT`.

Comment (googs1025) [reply pkg/controllers/leaderworkerset_controller.go]: done

File: pkg/controllers/leaderworkerset_controller.go (hunk 1 above)
Comment (googs1025) [reply pkg/controllers/leaderworkerset_controller.go]: done

**Review by kerthcet — COMMENTED** [review]
/lgtm
/approve

Thanks!

**Pull request merged**